server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -79,6 +81,7 @@ public NodesStatsRequest(StreamInput in) throws IOException {
             optionallyAddMetric(in.readBoolean(), Metric.ADAPTIVE_SELECTION.metricName());
         } else {
             requestedMetrics.addAll(in.readStringList());
+            restActionsFilters.addAll(in.readStringList());
```

Comment:
What happens if no filters are provided?

Reply:
if filters not provided, we'll get status count for all rest rest actions having count > 0